languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -8637,6 +8637,17 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
                 <token>fora</token>
                 <example>Nós aqui fora não concordamos</example>
             </antipattern>
+            <antipattern>
+                <token postag='AQ..S.+|NC.S.+' postag_regexp='yes'/>
+                <token postag='SPS00'/>
+                <token>nós</token>
```

Comment:
This makes sense, but I feel like it's not limited to 'nós'. Any plural personal pronoun could be substituted here. 'Para vocês', 'por eles', etc.